iXBRLViewerPlugin/iXBRLViewer.py

```diff
@@ -407,6 +407,13 @@ def createViewer(
         for n, report in enumerate(self.reports):
             self.footnoteRelationshipSet = ModelRelationshipSet(report, "XBRL-footnotes")
             self.currentTargetReport = self.newTargetReport(getattr(report, "ixdsTarget", None))
+            softwareCredits = {
+                match
+                for document in report.urlDocs.values()
```

Comment:
I think this is going to include creation software used for both the report and any taxonomy files (including filer extension and base taxonomy). We should limit this to just the inline report documents.